server/db/models/Product.js

```diff
@@ -1,12 +1,48 @@
 const { STRING } = require('sequelize')
 const db = require('../db')
 
+const Product = conn.define('product', {
```

Comment:
On line 4, it should db.define not conn.define.

Reply:
Ohh, thank you! I had  just changed it